src/release/upgrade.md

```diff
@@ -81,14 +135,9 @@ the Dart [`pub outdated` documentation]({{site.dart-site}}/tools/pub/cmd/pub-out
 $ flutter pub outdated
 ```
 
-## Keeping informed
-
-We publish breaking change announcements to the
-[Flutter announcements mailing list][flutter-announce].
-Aside from subscribing to receive announcements,
-we'd love to hear from you!
-
 [Flutter SDK archive]: {{site.url}}/release/archive
-[release channels]: {{site.repo.flutter}}/wiki/Flutter-build-release-channels
```

Comment:
Looks like this link definition is still reference on line 7 - https://github.com/flutter/website/blob/a2df8ed07998abedacca1f8bfc8296ac5b482382/src/release/upgrade.md?plain=1#L7

Perhaps with this extra information, the link can be dropped?

Reply:
oops. yeah i am trying to remove links to the wiki from this page. The goal is for the website to be our developer documentation and the wiki the contributor documentation.

Reply:
fixed

Reply:
I like that transition. Thanks!